kalite/distributed/static/js/distributed/content/views.js

```diff
@@ -95,7 +108,7 @@ window.ContentBaseView = BaseView.extend({
 
         this.active = false;
 
-        this.possible_points = 500;
+        this.possible_points = ds.distributed.turn_off_points_for_videos ? 0 : ds.distributed.points_per_video;
```

Comment:
Good catch. Maybe we'd want to separate these settings by content type, though? Or burn them with :fire:? Otherwise, I guess we're changing how many points a PDF is worth, etc?